Server/mods/deathmatch/logic/CPlayerManager.h

```diff
@@ -17,6 +17,7 @@ class CPlayerManager;
 #include "packets/CPacket.h"
 #include "CPlayer.h"
 #include "../Config.h"
+#include <string>
```

Comment:
it looks like `std::string` is defined already

Reply:
Thanks!